resources/assets/js/videos/components/videoScreen/videoPlayback.vue

```diff
@@ -167,7 +167,7 @@ export default {
         this.dummyCanvas.height = 1;
         this.video.addEventListener('play', this.setPlaying);
         this.video.addEventListener('pause', this.setPaused);
-        this.video.addEventListener('seeked', this.handleSeeked);
+        this.video.addEventListener('timeupdate', this.handleSeeked);
```

Comment:
This should still be `seeked`. The `timeupdate` event also fires while the video is playing so there will be redundand calls to `renderVideo()` here.